src/Post.js

```diff
@@ -63,35 +70,12 @@ class Post extends React.Component {
     this.setState({ updated_post: event.target.value });
   }
 
-  show() {
-    let token = Cookies.get("acebookSession")
-    fetch(`${BASE_URL}/api/v1/posts/` + this.state.post_id, {
-      method: 'GET',
-      headers: {
-        'Content-Type': 'application/json',
-        Accept: 'application/json',
-        Authorization: token
-      },
-      credentials: 'include',
-    })
-      .then(res => res.json())
-      .then(
-        (result) => {
-          this.setState({
-            post: result.message
-          });
-        })
-      .catch(err => console.log(err))
-  }
-
   render() {
     let edit_button = '';
     let delete_button = '';
-    // const post_id = this.props.data.id;
-    let edit_link = "#posts/edit/" + this.state.post_id
 
     if (this.props.data.editable && this.props.user.authCompleted) {
```

Comment:
if you're in the mindset of refactoring both this.props.data.editable, owned by and authCompleted could be defined at the beginning of render like done in app.js